### ISSUE TITLE
Link color contrast too low in Storymap content block

### ISSUE THREAD

**Issue by CourtneyLee333** [techmatters/terraso-product#512]
## Description
The color contrast between a link and the background of the content box is too low. The link is nearly invisible. 

## Steps To Reproduce

1. Go to URL: [Urbeapie y Paseo 13 storymap](https://app.terraso.org/tools/story-maps/611c3e42/urbeapie-y-paseo-13)
2. Click on Museo chapter
3. Scroll to other chapters with links
4. See the links with low contrast

## Expected behavior
The link has color that meets accessibility contrast requirements. 

## Actual behavior
The link color does not meet requirements. 

## Additional context
<img width="654" alt="Screenshot 2024-01-08 at 11 07 13 AM" src="https://github.com/techmatters/terraso-product/assets/8915145/cbc7b961-7c12-4f9a-9420-a827eef63039">
